### DISCUSSION TITLE
Xml Partial Response that is solved when refreshing the page

### DISCUSSION THREAD

**Discussion by dridi18** [primefaces/community#2581] [PrimeFaces] [answered]
Hello , a client of our application is facing the issue in the image below :
![partial1](https://github.com/user-attachments/assets/c9b13c31-1a4c-4814-a5f9-6067f3ad3c69)

when refreshing the page the issue is solved . But this issue occurs alot which made him raise an issue about it 

note that the client has keycloack( sso ) configuration . 
Is this related to the access token lifespan ? 
Thank you

**Comment by melloware** [comment, marked as answer]
It could be but its impossible to know for us just from the little you have provided.  

Please provide an executable example using the [PrimeFaces Test](https://github.com/primefaces/primefaces-test) project.  It is the only way developers can debug your problem to help.
